pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -165,163 +168,215 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 	nodeName := node.ObjectMeta.Name
 
 	// parse status & spec, create initial status object on first call
-	specAnnoData, statusAnnoData, err := r.parseAnnotations(node)
```

Comment:
`Reconcile()` is getting very large, time to break it up into some smaller functions

Reply:
broke it up into shorter pieces.